src/nemo_evaluator/orchestration/orchestrator.py

```
@@ -138,6 +139,14 @@ def _resolve_generation(config: EvalConfig, solver_cfg: Any) -> GenerationConfig
     return svc_gen
 
 
+def _resolve_request_timeout(service: Any) -> float:
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `service`

(ANN401)